spec/components/previews/support_interface/course_option_details_component_preview.rb

```diff
@@ -0,0 +1,24 @@
+module SupportInterface
+  class CourseOptionDetailsComponentPreview < ViewComponent::Preview
```

Comment:
👍 I'm glad we are adding more components to the previews. It's really helpful. Especially with multiple scenarios like this.